app/src/main/java/com/codepath/apps/critterfinder/fragments/PetDetailsFragment.java

```diff
@@ -61,5 +66,11 @@ private void setupPetDetailsView() {
                 //resize(DeviceDimensionsHelper.getDisplayWidth(getContext()), 0).
                 into(mPetImage);
         mPetDescription.setText(mPet.getDescription());
+        mShelterContactName.setText(mPet.getContactName());
+        mShelterContactPhone.setText(mPet.getContactPhone());
+        mShelterContactState.setText(", "+mPet.getContactState());
+        mShelterCity.setText(mPet.getContactCity());
+        mShelterEmail.setText(mPet.getContactEmail());
```

Comment:
maybe we should only set values on these strings if they aren't null. That would help us from seeing "null" written into the UI on our details screen.